sql-cli/sql_cli/__main__.py

```diff
@@ -79,7 +80,15 @@ def set_debug_mode(debug: bool) -> None:
     cls=AstroCommand,
     help="Print the SQL CLI version.",
 )
-def version() -> None:
+def version(
+    as_json: bool = typer.Option(
+        False, "--json", help="If the response should be in JSON format", show_default=True
+    ),
+) -> None:
+    if as_json:
+        output_dict = {"version": sql_cli.__version__}
+        print(json.dumps(output_dict))
+        return
     rprint("Astro SQL CLI", sql_cli.__version__)
```

Comment:
```suggestion
    if as_json:
        output_dict = {"version": sql_cli.__version__}
        print(json.dumps(output_dict))
    else:
        rprint("Astro SQL CLI", sql_cli.__version__)
```

I think it is nicer to use if/else here just to make it more clear that it only prints one or the other.